pkg/hintrunner/zero/zerohint_signature.go

```diff
@@ -297,7 +295,9 @@ func newDivModSafeDivHint() hinter.Hinter {
 				return err
 			}
 
-			return ctx.ScopeManager.AssignVariable("value", &value)
+			value_Big := new(big.Int).Set(&value)
```

Comment:
value is already a bigInt, the previous code was correct, it stores in scope a pointer to a bigInt with `&value`

Reply:
Right . Agreed.